app/models/portal/offering_embeddable_metadata.rb

```diff
@@ -0,0 +1,10 @@
+class Portal::OfferingEmbeddableMetadata < ActiveRecord::Base
```

Comment:
Since the reading and setting of the EmbeddableFilter is now sent through an API perhaps making this change is not as hard as it used to be. 🍀